proposed/2021/MachineReadableOutput.md

```diff
@@ -0,0 +1,807 @@
+
+```json
+{
+  "version": 1,
+  "parameters": "--include-transitive",
+  "projects": [
+    {
+      "path": "src/lib/MyProjectA.csproj",
+      "frameworks": [
+        {
+          "framework": "netcoreapp3.1",
+          "topLevelPackages": [
+              {
+                "id": "Microsoft.Extensions.Primitives",
+                "requestedVersion": "[1.0.0, 5.0.0]",
+                "resolvedVersion": "1.0.0"
+              },
+              {
+                "id": "NuGet.Commands",
+                "requestedVersion": "4.8.0-preview3.5278",
+                "resolvedVersion": "4.8.0-preview3.5278"
+              },
+              {
+                "id": "Text2Xml.Lib",
+                "requestedVersion": "[1.1.2, 2.0.0)",
+                "resolvedVersion": "1.1.2"
+              }
+          ],
+          "transitivePackages": [
+              {
+                "id": "Microsoft.CSharp",
+                "resolvedVersion": "4.0.1"
+              },
+              {
+                "id": "Microsoft.NETCore.Platforms",
+                "resolvedVersion": "1.1.0"
+              },
+      ...
+            ]
+        }
+      ]
+    },
+    {
+      "path": "src/lib/MyProjectB.csproj",
+      "frameworks": [
+        {
+          "framework": "netcoreapp3.1",
+          "topLevelPackages": [
+              {
+                "id": "NuGet.Commands",
+                "requestedVersion": "4.8.0-preview3.5278",
+                "resolvedVersion": "4.8.0-preview3.5278"
+              },
+              {
+                "id": "Text2Xml.Lib",
+                "requestedVersion": "1.1.2",
+                "resolvedVersion": "1.1.2"
+              }
+            ],
+            "transitivePackages": [
+              {
+                "id": "Microsoft.CSharp",
+                "resolvedVersion": "4.0.1"
+              },
+              {
+                "id": "Microsoft.NETCore.Platforms",
+                "resolvedVersion": "1.1.0"
+              },
+      ...
+            ]
+        },
+        {
+          "framework": "net5.0",
+          "topLevelPackages": [ 
+              {
+                "id": "NuGet.Commands",
+                "requestedVersion": "4.8.0-preview3.5278",
+                "resolvedVersion": "4.8.0-preview3.5278"
+              },
+              {
+                "id": "Text2Xml.Lib",
+                "requestedVersion": "1.1.2",
+                "resolvedVersion": "1.1.2"
+              }
+          ],
+          "transitivePackages": [
+              {
+                "id": "Microsoft.CSharp",
+                "resolvedVersion": "4.0.1"
+              },
+              {
+                "id": "Microsoft.NETCore.Platforms",
+                "resolvedVersion": "1.1.0"
+              },
+      ...
+            ]
+        }
+      ]
+    }
+  ]
+}
+```
+
+
+#### `> dotnet list package --include-transitive --outdated --framework net5.0`
+
+The following sources were used:
```

Comment:
formatting: this section needs to be fenced by triple backticks for consistent formatting

Reply:
Thank you for your review. Sure, I'll address it shortly.